apps/web/src/components/affine/workspace-setting-detail/panel/general/delete/index.tsx

```diff
@@ -35,8 +36,12 @@ export const WorkspaceDeleteModal = ({
   const { t } = useTranslation();
 
   const handleDelete = useCallback(() => {
-    onDeleteWorkspace();
-  }, [onDeleteWorkspace]);
+    onDeleteWorkspace().then(() => {
+      toast(t('Successfully deleted'), {
+        portal: document.body,
```

Comment:
The core is to mount the toast to the document.body. Because when we delete the workspace. The whole page will be refreshed and everything in React dom will rerender